rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -180,14 +180,17 @@ def test_exec(config):
             log.info("Testing asyc data notifications")
             ceph_version_id, _ = utils.get_ceph_version()
             if float(ceph_version_id[1]) >= 6 and float(ceph_version_id[5]) >= 8:
-                set_log = "ceph config set global log_to_file true"
-                out = utils.exec_shell_cmd(set_log)
                 cmd = " ceph orch ps | grep rgw"
                 out = utils.exec_shell_cmd(cmd)
                 rgw_process_name = out.split()[0]
                 utils.exec_shell_cmd(
                     f"ceph config set client.{rgw_process_name} rgw_data_notify_interval_msec 0"
                 )
+            ceph_conf.set_to_ceph_conf(
+                "global",
+                ConfigOpts.log_to_file,
```

Comment:
Here I think we need to define this parameter in:
ceph-qe-scripts/rgw/v2/lib/rgw_config_opts.py